test/src/org/omegat/tokenizer/TokenizerTest.java

```diff
@@ -188,6 +188,7 @@ public void testChinese() {
      * @see <a href=
      *      "https://groups.yahoo.com/neo/groups/OmegaT/conversations/messages/28395">
      *      User group discussion</a>
+     * @see <a href="https://sourceforge.net/p/omegat/mailman/message/36839317/">Sourceforge archive</a>
```

Comment:
We can remove yahoo URL here, too.

Reply:
Sure, but this one I was not 100% sure that it was pointing at the right mail.

Reply:
You're right. I just did not feel comfortable removing information. If you think it is not relevant, I am totally fine with your decision.